nsv13/code/modules/overmap/weapons/plasma_gun.dm

```diff
@@ -0,0 +1,465 @@
+/obj/machinery/ship_weapon/plasma_caster
+	name = "\improper Magnetic Phoron Acceleration Caster"
+	icon = 'nsv13/icons/obj/plasma_gun.dmi' //This sprite cost me $150
```

Comment:
Given we generally don't want anything paid in this codebase etc as we are or should be all volunteers.
We take the approach of don't ask, don't tell, so please don't add a price tag here, joke or not.

Reply:
Removed comment